gridcomps/History/MAPL_HistoryGridComp.F90

```diff
@@ -1908,6 +1924,7 @@ subroutine Initialize ( gc, import, dumexport, clock, rc )
 
       end if
 
+!***----------------------------------------
```

Comment:
Does this comment add value?

Reply:
I left a marker for a big block.  I have deleted now.